### PR TITLE
Fixed Excl. tax location so it won't overflow

### LINKED ISSUE #4293: Price excl.tax is shifted left for shipping method if shipping method has long name
**Description**:
Price excl.tax is shifted left for shipping method if shipping method has long name 

**Expected behavior**:
Delivery name and price is alligned 

**To Reproduce**:
**Admin panel:**
1. Go to Stores -> Configuration -> Sales -> Delivery method
2. Change title and method name for shipping method X
3. Save
4. Clear cache

**FE:**
1. Open PDP
2. Add product to cart
3. Go to chipping step
4. Scroll down to Shipping methods

**Actual result**:
Price excl.tax is shifted left
Screenshot 2022-02-15 at 09 00 53

### PR DESCRIPTION
**Pull request by SalimMersally**
**Related issue(s):**
* Fixes #4293 

**Problem:**
* [Price excl.tax is shifted left for shipping method if shipping method has long name

**In this PR:**
* FIxed Excl. tax postion so it won't overflow its container

### CODE REVIEW

**Comment by aleksandrakorolova** [comment]
Issue fixed but Please, return price excl tax on the previous place if method name is not long 
Screenshot 2022-03-16 at 10 44 31

File: packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.component.js [hunk 1]
```diff
@@ -38,12 +38,49 @@ export class CheckoutDeliveryOption extends PureComponent {
...
     };
 
+    state = { isOverflowed: false };
+
+    rowRef = createRef();
+
+    subPriceRef = createRef();
+
+    componentDidMount() {
+        if (this.rowRef.current && this.subPriceRef.current) {
```
Comment (riha112) [inline packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.component.js:48]: This if block can be replaced with `updateState()` function call, as they do the same thing.

File: packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.component.js [hunk 2]
```diff
@@ -38,12 +38,49 @@ export class CheckoutDeliveryOption extends PureComponent {
...
+    }
+
+    componentWillUnmount() {
+        window.removeEventListener('resize', () => this.updateState());
+    }
+
+    updateState() {
+        if (this.rowRef.current && this.subPriceRef.current) {
+            const rowLeft = this.rowRef.current.getBoundingClientRect().left;
+            const SubPriceLeft = this.subPriceRef.current.getBoundingClientRect().left;
```
Comment (riha112) [inline packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.component.js:66]: Reduce access chaining to something like:
```js
const { current: rowEl } = this.rowRef;
const { current: subPriceEl } = this.subPriceRef;

if (rowEl && subPriceEl) {
    const rowLeft = rowEl.getBoundingClientRect().left;
    const SubPriceLeft = subPriceEl.getBoundingClientRect().left;
    ...
```

**Review by riha112 — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.component.js [hunk 3]
```diff
@@ -38,12 +38,45 @@ export class CheckoutDeliveryOption extends PureComponent {
...
+        window.removeEventListener('resize', () => this.updateState());
+    }
+
+    updateState() {
+        const { current: rowEl } = this.rowRef;
+        const { current: subPriceEl } = this.subPriceRef;
+
+        if (rowEl && subPriceEl) {
+            const rowLeft = rowEl.getBoundingClientRect().left;
+            const SubPriceLeft = this.subPriceRef.current.getBoundingClientRect().left;
```
Comment (riha112) [inline packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.component.js:62]: rename `SubPriceLeft ` to `subPriceLeft`
change `this.subPriceRef.current.` to `subPriceEl.`

**Review by riha112 — CHANGES_REQUESTED** [review]

**Review by riha112 — APPROVED** [review]

**Pull request merged**